### PR TITLE
Fixing up bugs related to missing items in schema or output values mi…

### PR DESCRIPTION
**Pull request by shainw**
…ssing from query

Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml [hunk 1]
```diff
@@ -39,5 +39,5 @@ query: |
     | summarize FailedLoginsCountInRunTime = count() by User = Identity 
   ) on User 
   | where FailedLoginsCountInRunTime > LearningThreshold
-  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
```
Comment (ashwin-patil) [inline Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml:42]: Is entity extraction supposed to be commented ? 
I didn`t see any instructions when to use or not use in description

File: Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml [hunk 2]
```diff
@@ -33,4 +33,4 @@ query: |
     | distinct Country, Actor 
   ) on Actor 
   | where set_Country !contains Country
-  | extend AccountCustomEntity = Actor, IPCustomEntity = IPaddress
+  //| extend AccountCustomEntity = Actor, IPCustomEntity = IPaddress
```
Comment (ashwin-patil) [inline Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml:36]: same feedback on commenting

File: Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml [hunk 3]
```diff
@@ -24,6 +24,6 @@ query: |
   | where ResultType == 0
   | summarize CountOfLocations = dcount(Location), Locations = make_set(Location) by User = Identity
   | where CountOfLocations > 1
-  | extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
+  //| extend AccountCustomEntity = UserPrincipalName, IPCustomEntity = IPAddress
```
Comment (ashwin-patil) [inline Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml:27]: same feedback on commenting

File: Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml (hunk 1 above)
Comment (shainw) [reply Detections/SigninLogs/Brute Force Attack against GitHub Account.yaml:42]: Yeah, this is broken so commented it out while the author fixes it.

File: Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml (hunk 2 above)
Comment (shainw) [reply Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml:36]: Yeah, this is broken so commented it out while the author fixes it.

File: Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml (hunk 3 above)
Comment (shainw) [reply Detections/SigninLogs/Sign-in Burst from Multiple Locations.yaml:27]: Yeah, this is broken so commented it out while the author fixes it.

**Pull request merged**